pyproject.toml

```diff
@@ -1,4 +1,6 @@
 # Only a configuration storage, for now
+[tool.aliases]
+test = "pytest"
```

Comment:
FWIW, is that a needed thing anymore? 😕

Reply:
Not sure what the purpose is tbf. I'm just keeping feature parity.